projects/vdk-control-cli/src/vdk/internal/control/command_groups/job/deploy_cli.py

```diff
@@ -50,13 +50,11 @@ class DeployOperation(Enum):
 # Deploy job and wait until deployed
 new_version=`vdk deploy -p /home/user/data-jobs/example-job -o json | jq '.job_version'`
 while ! vdk deploy --show -o json -n example-job -t job-team | grep $new_version ; do echo "waiting ..."; sleep 10; done
-
 \b
-# Deploy multiple Data Jobs in a single command
-echo "Job1
-job2
-job3" | xargs -I {JOB} vdk deploy  -t job-team  -n {JOB} –p <PARENT_DIR_TO_JOBS>/{JOB}
-
+# Deploy multiple Data Jobs in a single command. Data jobs must be created and
+reside in their own folders with the same name as the data-job's names e.g. job1
+these folders should be within a <PARENT_DIR_TO_JOBS> folder:
+echo "job1\n job2\n job3" | xargs -I "#" vdk deploy -t job-team -n "#" -p "<PARENT_DIR_TO_JOBS>/#" -r "Deploy reason."
```

Comment:
Wouldn't it be clearer to set the text for the reason as `"<reason-for-deployment>"`?